### ISSUE TITLE
Pseudo-patterns do not work in PL 2.11.0 and above

### ISSUE THREAD

**Issue by danwhite85** [pattern-lab/patternlab-node#711]
Pseudo-patterns are not working as expected in PL 2.11.0 and above. I've tested 2.10.0, 2.11.0, 2.11.1, and 2.12.0. 

In all versions over 2.10.0 the pseudo-pattern data is available in all pattern instances. In the example found at [http://patternlab.io/docs/pattern-pseudo-patterns.html](url) the emergency message would always be shown.

_homepage.mustache_
```
<div id="main-container">
    {{# emergency }}
        <div class="emergency">Oh Noes! Emergency!</div>
    {{/ emergency }}
    { ...a bunch of other content... }
</div>
```

_00-homepage~emergency.json_
```
{
    "emergency": true
}
```

**Comment by bmuenzenmeyer** [comment]
Hey @danwhite85 

Thanks so much for the thorough report. Sorry for the regression. I thought my pseudo-pattern test coverage was enough but there is clearly a gap somewhere

**Comment by bmuenzenmeyer** [comment]
Doing some sleuthing 🕵️ 

https://github.com/pattern-lab/patternlab-node/pull/693/files seems to be a culprit 🐛

**Comment by tburny** [comment]
Nice fix

**Comment by acarnwath** [comment]
@bmuenzenmeyer am I correct in my understanding that this was fixed on the 3.0.0.alpha branch?  If we do not want to be on alpha, we would need to revert back to < 2.11?

**Comment by bmuenzenmeyer** [comment]
this was fixed in alpha yes
